ci/do_ci.sh

```diff
@@ -85,11 +85,19 @@ CI_TARGET=$1
 
 if [[ $# -gt 1 ]]; then
   shift
+  # If this is a fuzz test, add _with_libfuzzer
+  if [[ "$CI_TARGET" == "bazel.fuzz" ]]; then
+    FUZZ_TEST_TARGETS=$*"_with_libfuzzer"
+  fi
   TEST_TARGETS=$*
 else
   TEST_TARGETS=//test/...
+  if [[ "$CI_TARGET" == "bazel.fuzz" ]]; then
+    FUZZ_TEST_TARGETS="$(for t in $(bazel query "attr('tags','libfuzzer',//test/...)"); do echo $t; done;)"
```

Comment:
Do you need the `for` loop? Doesn't `bazel query` give us one per line, which is effectively a bash list?